addons/mail/static/src/new/core/thread_service.js

```diff
@@ -842,6 +845,28 @@ export class ThreadService {
             direction: "none",
         });
     }
+
+    /**
+     * @param {import("@mail/new/core/thread_model").Thread} thread
+     * @param {import('@mail/new/core/persona_model').Persona} persona
+     */
+    avatarUrl(thread, persona) {
+        if (!persona) {
+            return DEFAULT_AVATAR;
+        }
+        if (thread?.model === "mail.channel") {
+            return `/mail/channel/${thread.id}/${persona.type}/${persona.id}/avatar_128`;
```

Comment:
note that current code allows user to pass through but the python does not support it